src/System.Buffers/src/project.json

```diff
@@ -1,11 +1,16 @@
 {
+  "compilationOptions": {
+    "allowUnsafe": true
+  },
   "dependencies": {
-    "System.Diagnostics.Debug": "4.0.0", 
+    "System.Diagnostics.Debug": "4.0.0",
+    "System.Diagnostics.Tools": "4.0.1-beta-23409",
     "System.Reflection.Primitives": "4.0.0",
     "System.Resources.ResourceManager": "4.0.0",
     "System.Runtime": "4.0.0",
     "System.Runtime.Extensions": "4.0.0",
     "System.Runtime.InteropServices": "4.0.0",
+    "System.Slices": "0.1.0-d103015-1",
```

Comment:
Like I explained above, in order to do that we need to enable custom directory structure in the project.json, which is why I didn't do it like that in the first place. Also, by talking to Wes, he said that having no dependencies outside of your folder is a goal, and all external dependencies should happen through nuget packages.